pyartcd/pyartcd/pipelines/ocp4_scan_konflux.py

```
@@ -277,41 +277,45 @@ async def handle_bridge_bug_mirroring(self):
+                f"--working-dir={self._elliott_working}",
+                f"--data-path={self.data_path}",
+                "find-bugs:bridge-mirror",
+            ]
+            if self.runtime.dry_run:
+                cmd.append("--dry-run")
 
-        try:
             await exectools.cmd_assert_async(cmd)
-        except ChildProcessError as e:
+
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)